node/src/cli.rs

```diff
@@ -37,6 +40,48 @@ pub enum Cli {
         #[arg(long, default_value = "8192")]
         desired_presignatures_to_buffer: usize,
     },
+    MigrateKeyshare(MigrateKeyshare),
+}
+
+#[derive(Parser, Debug)]
+pub struct MigrateKeyshare {
+    #[arg(long, env("MPC_HOME_DIR"))]
+    pub home_dir: String,
+    #[arg(env("GCP_KEYSHARE_SECRET_ID"))]
+    pub gcp_keyshare_secret_id: String,
+    #[arg(env("GCP_PROJECT_ID"))]
+    pub gcp_project_id: String,
+    /// Hex-encoded 16 byte AES key for local storage encryption.
+    /// This key should come from a secure secret storage.
+    #[arg(env("MPC_SECRET_STORE_KEY"))]
+    pub secret_store_key_hex: String,
+}
+
+impl MigrateKeyshare {
+    async fn migrate_key_shares(self) -> Result<(), anyhow::Error> {
+        let gcp_key_share_storage = KeyshareStorageFactory::Gcp {
+            project_id: self.gcp_project_id,
+            secret_id: self.gcp_keyshare_secret_id,
+        }
+        .create()
+        .await?;
+
+        let encryption_key = AesEncryptionKey::try_from(self.secret_store_key_hex.as_str())?;
+
+        let local_key_share_storage = KeyshareStorageFactory::Local {
+            home_dir: self.home_dir.into(),
+            encryption_key: encryption_key,
```

Comment:
tip: Run "cargo clippy" to catch style suggestions like these (encryption_key alone is enough)